utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -16,11 +16,12 @@ use std::task::Poll;
 use std::time::Duration;
 use std::time::Instant;
 
+use bytesize::ByteSize;
 use strum::AsStaticRef;
 
-const MEBIBYTE: usize = 1024 * 1024;
-const MEMORY_LIMIT: usize = 512 * MEBIBYTE;
-const MIN_MEM_USAGE_REPORT_SIZE: usize = 100 * MEBIBYTE;
+static MEMORY_LIMIT: u64 = 512 * bytesize::MIB;
```

Comment:
Worth sending a PR to `bytesize` with `const`s to make`static MEMORY_LIMIT: ByteSize = ByteSize::mib(512);` work.(obv no need to block PR overt this)

Reply:
```
error[E0015]: calls in statics are limited to constant functions, tuple structs and tuple variants
  --> utils/near-performance-metrics/src/stats_enabled.rs:22:34
   |
22 | static MEMORY_LIMIT2: ByteSize = ByteSize::mb(512);
   |                                  ^^^^^^^^^^^^^^^^^

```

Reply:
Also `+=` operator is missing

Reply:
https://github.com/hyunsik/bytesize/pull/21